### ISSUE TITLE
document which packages are opsoleted by tokamak

### ISSUE THREAD

**Issue by flying-sheep** [vertexclique/tokamak#26]
so i found out i can disable

* language-rust
* terminal-plus
* build-cargo

but not

* linter-rust

by trial and error. it would be nice to have this on the webpage or readme

**Comment by vertexclique** [comment]
Can you please clarify this with what did you see as error?
Currently I had disabled linter-rust for example and it doesn't do linting?

Probably you don't want to see the statusbar at the bottom after disabling `linter-rust` right?
If it is like that `linter` is a atom plugin that serves as a plugin base for linting multiple languages with itself. First you can disable `linter-rust` then `linter` maybe? This way can be your solution.

**Comment by colin-kiegel** [comment]
I think the question:

Which other rust packages are redundant with tokamak and which can be used complementary?

At least that is what I was wondering too. So a short Statement like "you might want to combine this package with X, but you won't need Y" would help because other people might be wondering which packages they need.

**Comment by flying-sheep** [comment]
> Which other rust packages are redundant with tokamak and which can be used complementary?

exactly. tokamak includes a rust grammar, so also installing `language-rust` is redundant and you end up with two selectable rust grammars.

so i would like tokamak to list packages that you can disable without losing functionality after installing tokamak

**Comment by vertexclique** [comment]
OK the thing is `linter-rust` shows `language-rust` as dependency.
Yeah and we don't need grammar this is completely true, but we have contributed to original grammar by adding some fields which is not in `language-rust`.
https://github.com/AtomLinter/linter-rust/blob/master/package.json

So there are two options for that:
* Use our own grammar - fork linter-rust and modify it for not using language-rust as package-dep
* Use default grammar supplied by language-rust

Your choice, your style.

**Comment by vertexclique** [comment]
@colin-kiegel 
I couldn't understand what you mean by 
> you might want to combine this package with X, but you won't need Y

and redundant and complementary part. Redundant part is probably grammar part and complementary part is rest of it. Because if I exclude `linter` there will be no linter as you can think of.

**Comment by flying-sheep** [comment]
in the first comment: `language-rust`, `terminal-plus`, `build-cargo` are redundant; `linter-rust` is complementary

**Comment by colin-kiegel** [comment]
Isn't it possible to contribute the grammar changes to `language-rust` upstream?

If that works out, tokamak could define language-rust as a dependency. Adding linter-rust wouldn't do any harm then.

**Comment by flying-sheep** [comment]
huh? how does it do harm now?

**Comment by colin-kiegel** [comment]
sorry for the bad choice of words: Adding linter-rust wouldn't **install any redundant dependencies** then.

Better? :-)

**Comment by vertexclique** [comment]
Long time no see! Sorry for that. I will update repo wiki with this information too. Thanks for feedback.
